### ISSUE TITLE
ENH: Iris image imort/export

### ISSUE THREAD

**Issue by cpelley** [SciTools/iris-extras#6]
I propose here to bring over my PR to iris to iris-extras.
https://github.com/SciTools/iris/pull/1727

This way I can have it reviewed by colleagues or someone who has an invested interested interest in using it.  Following that, we can merge.

If I hear no objection on this, I'll put a PR together pointing to iris-extras.
I thought it best to raise an issue on this first so my PR does not come as a surprise.

**Context:**

The ancillary work happens to require some source files read which are in BIL format.  Ideally I would like this capability shared with the iris community (hence this issue).  The existing iris experimental geotiff should also be moved at some point to here (if this issue goes ahead).

**Comment by rhattersley** [comment]
> If I hear no objection on this, I'll put a PR together pointing to iris-extras.

:+1: Yes please. :smile: 

Do you have a plan already for how to structure this extension? e.g. A self-contained library in its own sub-directory (with its own `setup.py`)?

**Comment by pelson** [comment]
> A self-contained library in its own sub-directory (with its own setup.py)?

:+1: - maybe we can just create a repo for it? ```iris-image-io``` or something?

**Comment by rhattersley** [comment]
> :+1: - maybe we can just create a repo for it? iris-image-io or something?

Yeah, @niallrobinson's [Cube Explorer PR](https://github.com/SciTools/iris-extras/pull/5) had got me thinking the same thing - a new repo might be a better fit.

**Comment by cpelley** [comment]
Thank you both, that's great news.  I'm happy to go with a new repo, cheers
Sure, having its own setup.py will probably be a must I should think.

**Comment by cpelley** [comment]
^On the basis of the above comments, perhaps this is the time to suggest that `iris-extras` is renamed?
Perhaps to `iris-regrid` or something.
